access/templates/access/_error_summary.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+
+<div class="error-summary">
+	<div class="background">
+		<div class="text-container">
```

Comment:
Does this really require three nested divs to get correct positioning etc? Seems strange as I would expect that two should be sufficient, but maybe I'm wrong.

Reply:
This was done based on earlier suggestions in the other repo, but two will do as well, so the change has been made. Just let me know then if there is something else here that would require changing or thinking about. Or alternatively whether this pr has become squashable :)